styles.css

```diff
@@ -420,17 +444,20 @@ main {
   background-color: #2230d2;
 }
 
-@media (min-width: 400px) {
+@media (min-width: 24rem) {
   body {
-    width: 1640px;
+    width: 117.2242rem;
   }
```

Comment:
- [ ] All the sections of your desktop page are good and very similar to the Figma template, but they are not responsive and the user screen needs to be very large to show the whole content without the page cutting out the content. Kindly reduce the width of some components in all the sections and adjust some horizontal margins and paddings to make the content fit on any desktop screen without the need of a horizontal scroll, for a better user experience.
 - Your design(1200px wide):
 - 
<img width="1204" alt="image" src="https://user-images.githubusercontent.com/61919617/221016809-97edc4ef-f410-4b88-b266-82671e8dcfc0.png">
<img width="1202" alt="image" src="https://user-images.githubusercontent.com/61919617/221016940-9431a72c-faee-4aaa-94dd-8f12d0e65f2e.png">